packages/strapi-plugin-content-manager/oas.yml

```diff
@@ -0,0 +1,670 @@
+openapi: 3.0.2
```

Comment:
`PUT /content-manager/collection-types/{model}/{id}`: the body param is not present